### PR TITLE
RMAC-8338: Fix crash loading runtime

### PR DESCRIPTION
**Pull request by cangosta**

### CODE REVIEW

File: CefGlue.Common/CefRuntimeLoader.cs [hunk 1]
```diff
@@ -30,12 +30,13 @@ private static void InternalInitialize(CefSettings settings = null, KeyValuePair
 
             settings.UncaughtExceptionStackSize = 100; // for uncaught exception event work properly
 
-            var probingPaths = GetSubProcessPaths();
-            var path = probingPaths.FirstOrDefault(p => File.Exists(p));
-            if (path == null)
+            var basePath = AppContext.BaseDirectory;
+            var probingPaths = GetSubProcessPaths(basePath);
+            var subProcessPath = probingPaths.FirstOrDefault(p => File.Exists(p));
```
Comment (joaompneves) [inline CefGlue.Common/CefRuntimeLoader.cs:35]:
```suggestion
            var probingPaths = GetSubProcessPaths();
            var subProcessPath = probingPaths.Select(p => Path.Combine(p, BrowserProcessFileName)).FirstOrDefault(p => File.Exists(p));
```

File: CefGlue.Common/CefRuntimeLoader.cs [hunk 2]
```diff
@@ -44,7 +45,7 @@ private static void InternalInitialize(CefSettings settings = null, KeyValuePair
                     break;
 
                 case CefRuntimePlatform.MacOS:
-                    var resourcesPath = Path.Combine(path, "Resources");
+                    var resourcesPath = Path.Combine(basePath, "Resources");
```
Comment (joaompneves) [inline CefGlue.Common/CefRuntimeLoader.cs:48]:
```suggestion
                    var resourcesPath = Path.Combine(subProcessPath, "Resources");
```

File: CefGlue.Common/CefRuntimeLoader.cs [hunk 3]
```diff
@@ -53,8 +54,8 @@ private static void InternalInitialize(CefSettings settings = null, KeyValuePair
                     settings.NoSandbox = true;
                     settings.MultiThreadedMessageLoop = false;
                     settings.ExternalMessagePump = true;
-                    settings.MainBundlePath = path;
-                    settings.FrameworkDirPath = path;
+                    settings.MainBundlePath = basePath;
+                    settings.FrameworkDirPath = basePath;
```
Comment (joaompneves) [inline CefGlue.Common/CefRuntimeLoader.cs:58]:
```suggestion
                    settings.MainBundlePath = subProcessPath;
                    settings.FrameworkDirPath = subProcessPath;
```

File: CefGlue.Common/CefRuntimeLoader.cs (hunk 2 above)
Comment (cangosta) [reply CefGlue.Common/CefRuntimeLoader.cs:48]: but the Resources folder is not under the subProcessPath, that is why we were having a failure

**Review by joaompneves — APPROVED** [review]

**Pull request merged**